pkg/apis/certificates/types.go

```diff
@@ -49,6 +49,12 @@ type CertificateSigningRequestSpec struct {
 	// `scope-hostname.io/name`.
 	// Distribution of trust for signers happens out of band.
 	// You can select on this field using `spec.signerName`.
+	// Kubernetes provides built-in signers that each have a well-known signerName:
+	// 1. kubernetes.io/kube-apiserver-client
+	// 2. kubernetes.io/kube-apiserver-client-kubelet
+	// 3. kubernetes.io/kubelet-serving
+	// 4. kubernetes.io/legacy-unknown
+	// Custom signerNames can also be specified and that those are external signers and as such the control plane signer will not issue certificates.
 	SignerName string
```

Comment:
this is already documented on the v1 struct:

https://github.com/kubernetes/kubernetes/blob/184c6d348c80c1f529b742c8959677817689a182/staging/src/k8s.io/api/certificates/v1/types.go#L64-L85

I'd suggest copying the documentation from there if we want to also update this field's doc

Reply:
Thanks. I will revise it